src/cmd/misc/ship/ship.js

```diff
@@ -21,8 +21,8 @@ exports.ship = {
         }
         
         if (ctx.mentions[0] && !ctx.mentions[1] && new RegExp(`<@!?${ctx.mentions[0].id}>`).test(ctx.args[0])) {
-            ctx.mentions[0] = ctx.mentions[0];
             ctx.mentions[1] = null;
+            ctx.mentions[0] = ctx.mentions[1];
```

Comment:
Can you put this on another CL? You're changing way beyond the scope of your PR.